src/Root.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage, TabsPage, PageNotFound } from './pages';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to=".." />} />
```

Comment:
The navigation from '/home' to '/' is not implemented correctly. It is currently set to navigate to the parent directory, which may not necessarily be '/'. You should navigate to '/' directly.